blueprints/addon/files/.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
+  # - EMBER_TRY_SCENARIO=ember-source
```

Comment:
If we don't add it (since we removed `default`) we will never _actually_ run tests with ember-source in CI...